include/libtorrent/create_torrent.hpp

```diff
@@ -366,8 +366,7 @@ namespace libtorrent {
 		bool m_include_symlinks:1;
 	};
 
-namespace detail {
-
+	namespace detail {
```

Comment:
I'm actually hoping to go the other direction, where namespaces don't add additional indentation. (in some way to adopt the of C++17 feature of being able to open nested namespaces in a single namespace definition)